cmd/ocsp-responder/main.go

```diff
@@ -223,104 +245,132 @@ func (src *dbSource) Response(ctx context.Context, req *ocsp.Request) ([]byte, h
 
 	// If the primary source returns first, check the output and return
 	// it. If the secondary source wins, then wait for the primary so the
-	// results from the seconary can be verified. It is important that we
+	// results from the secondary can be verified. It is important that we
 	// never return a response from the redis source that is good if mysql
 	// has a revoked status. If the secondary source wins the race and
-	// passes these checks, return it's response instead.
+	// passes these checks, return its response instead.
 	select {
 	case <-ctx.Done():
 		err := fmt.Errorf("looking up OCSP response for serial: %s err: %w", serialString, ctx.Err())
-		src.log.Debugf(err.Error())
-		ocspLookups.WithLabelValues("canceled").Inc()
+		src.log.Debugf("%s", err)
```

Comment:
Through this section you have a bunch of places where you `Debugf` or `AuditErrf` and error, and then return that error. In `ocsp/responder.go`, there's already code that will `Infof` any error returned from Response. I think we should remove most of the logging calls in this function in favor of that `Infof`. Were you getting doubled-up errors during testing?

Reply:
Removed unnecessary log lines. Thanks for the help!